### ISSUE TITLE
vital.ht units

### ISSUE THREAD

**Issue by dckc** [SCILHS/i2p-transform#5]
The spec says HT is "Height (in inches)" but our EHR seems to report cm. I don't see any support for units in this code. Is this not an issue because you have inches in your i2b2 fact table?

I suppose i2b2 can express conversion factors in c_metadataxml, but I don't look forward to writing code to handle that.

Suggestions?

**Comment by lcphillips2** [comment]
Dan

The conversion should be in the c_metadataxml already.
Or am I missing something?

Lori

**Comment by dckc** [comment]
The conversion factor can be expressed in c_metadataxml, but the `PCORNetVital` procedure in this CDM ETL code doesn't consult it.

https://github.com/SCILHS/i2p-transform/blob/master/Oracle/PCORNetLoader_ora.sql#L1092

**Comment by dckc** [comment]
cc @njgraham @bos666

**Comment by lcphillips2** [comment]
Got it..

**Comment by jklann** [comment]
Yes, I'm not looking forward to writing that code either. <g>

See in the [documentation](https://github.com/SCILHS/i2p-transform/blob/master/PopMedNet-i2b2%20Transform%20Documentation.md), "Vitals unit conversions - v6 assumes the fact table uses the units PCORI is expecting."

Note that one reason we lowered the priority on this is that if sites use inconsistent units, unit conversions must be turned on in the CRC. If that is the case, EVERY fact that supports units must have the units column filled in with a value in the MetadataXML, or data is lost during i2b2 queries. So given the state of unit conversions, we decided it's safer to just make all our sites standardize their units to PCORI's expectations.
